src/main/java/org/rev317/min/callback/MessageCallback.java

```diff
@@ -9,4 +9,9 @@ public static final void messageListenerHook(int type, String name, String messa
         final MessageEvent messageEvent = new MessageEvent(type, name, message);
         ScriptEngine.getInstance().dispatch(messageEvent);
     }
+
+    public static final void messageListenerHook(int type, String name, String message, String title) {
```

Comment:
Why don't you remove the other method, as this is Dreamscape specific only anyway.